extra/renderer-and-libwebrtc-tests.js

```diff
@@ -442,15 +442,10 @@ function TestMediaRenderCatpure(videoEl) {
 	});
 }
 
-var testMediaRenderCatpureAnimateFrame;
-function TestMediaRenderCatpureAnimate() {
-	TestMediaRenderCatpure(peerVideoEl);
-	testMediaRenderCatpureAnimateFrame = requestAnimationFrame(TestMediaRenderCatpureAnimate);
-}
-
 // Disabled to avoid confusion with remoteStream
 var cloneStream;
 var cloneVideoEl;
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 function TestPluginMediaStreamClone(mediaStream) {
```

Comment:
This one also appears unused, but I left it since there is a comment saying it was intentionally disabled.

Reply:
Same this is here for testing